R/render.R

```diff
@@ -37,20 +37,25 @@
 #' }
 #'
 #' @rdname render
-render_book <- function(project_path = ".", site_url = NULL, profile = NULL) {
-  render(path = project_path, site_url = site_url, type = "book", profile = profile)
+render_book <- function(project_path = ".",
+                        site_url = NULL, profile = NULL) {
+  render(project_path, site_url = site_url, type = "book", profile = profile)
 }
 
 #' @export
 #' @rdname render
-render_website <- function(project_path = ".", site_url = NULL, profile = NULL) {
-  render(path = project_path, site_url = site_url, type = "website", profile = profile)
+render_website <- function(project_path = ".",
+                           site_url = NULL, profile = NULL) {
+  render(project_path, site_url = site_url, type = "website", profile = profile)
 }
 
-render <- function(path = ".", site_url = NULL, type = c("book", "website"), profile) {
+render <- function(path = ".",
+                   site_url = NULL,
+                   type = c("book", "website"),
+                   profile = NULL) {
   # configuration ----
   config <- file.path(path, "_quarto.yml")
-  config_contents <- yaml::read_yaml(config, handlers = list(seq = function(x) x))
+  config_contents <- read_yaml(config) # nolint: object_usage_linter
```

Comment:
Same as suggested in `register.R`.

```suggestion
  config_contents <- read_yaml(config)
```